vmdb/app/controllers/application_controller.rb

```diff
@@ -2168,6 +2168,10 @@ def prov_redirect(typ=nil)
         render_flash_not_applicable_to_model(typ)
         return
       end
+      if typ == "migrate" && !VmOrTemplate.supports_migrate?(vm_ids)
```

Comment:
I'm not sure I understand your comment.  `is_available?`  would tell you if any of this is applicable or not right?